### ISSUE TITLE
OLM instance creation contains missing parameters within the yaml view

### ISSUE THREAD

**Issue by bfabec** [openshift/console#6399]
In OCP 4.5.5, when creating an instance within an operator (OLM), there are two views: yaml and form.  The form view is rendering correctly, but the yaml view is missing properties that have empty default values. This is new in OCP 4.5.x, and we would still like to render properties with empty values as it allows the user to see which properties needs values. This problem/change renders the yaml editor unusable. 

CSV (note clusterDomain):
```
apiVersion: operators.coreos.com/v1alpha1
kind: ClusterServiceVersion
metadata:
  annotations:
    alm-examples: |-
      [
        {
          "apiVersion": "noi.ibm.com/v1beta1",
          "kind": "NOI",
          "metadata": {
            "name": "example-noi"
          },
          "spec": {
            "advanced": {
              "antiAffinity": false,
              "imagePullPolicy": "IfNotPresent",
              "imagePullRepository": "cp.icr.io/cp/noi"
            },
            "clusterDomain": "",
            "deploymentType": "trial",
            "entitlementSecret": "",
            "ingress": {
              "class": "nginx",
              "enabled": false,
              "prefix": "nginx.ingress.kubernetes.io"
            },
            "integrations": {
              "humio": {
                "url": ""
              }
            },
            "ldap": {
              "baseDN": "dc=mycluster,dc=icp",
              "bindDN": "cn=admin,dc=mycluster,dc=icp",
              "groupFilter": "cn=%s,ou=groups",
              "mode": "standalone",
              "port": "389",
              "serverType": "CUSTOM",
              "sslPort": "636",
              "storageClass": "",
              "storageSize": "1Gi",
              "suffix": "dc=mycluster,dc=icp",
              "url": "ldap://localhost:389",
              "userFilter": "uid=%s,ou=users"
            },
            "license": {
              "accept": false
            },
            "persistence": {
              "enabled": false,
              "storageClassCassandraBackup": "",
              "storageClassCassandraData": "",
              "storageClassCouchdb": "",
              "storageClassDB2": "",
              "storageClassElastic": "",
              "storageClassImpactGUI": "",
              "storageClassImpactServer": "",
              "storageClassKafka": "",
              "storageClassNCOBackup": "",
              "storageClassNCOPrimary": "",
              "storageClassZookeeper": "",
              "storageSizeCassandraBackup": "50Gi",
              "storageSizeCassandraData": "50Gi",
              "storageSizeCouchdb": "5Gi",
              "storageSizeDB2": "5Gi",
              "storageSizeElastic": "75Gi",
              "storageSizeImpactGUI": "5Gi",
              "storageSizeImpactServer": "5Gi",
              "storageSizeKafka": "50Gi",
              "storageSizeNCOBackup": "5Gi",
              "storageSizeNCOPrimary": "5Gi",
              "storageSizeZookeeper": "5Gi"
            },
            "serviceContinuity": {
              "continuousAnalyticsCorrelation": false,
              "enableAnalyticsBackups": false,
              "isBackupDeployment": false
            },
            "topology": {
              "appDisco": {
                "certSecret": "",
                "dbsecret": "",
                "dburl": "",
                "dsDefaultExistingClaim": "",
                "dsStorageClass": "",
                "dsStorageSize": "5Gi",
                "enabled": false,
                "pssExistingClaim": "",
                "pssStorageClass": "",
                "pssStorageSize": "5Gi",
                "secure": false,
                "sssStorageClass": "",
                "sssStorageSize": "5Gi"
              },
              "enabled": true,
              "netDisco": false,
              "observers": {
                "alm": false,
                "ansibleawx": false,
                "appdynamics": false,
                "aws": false,
                "azure": false,
                "bigfixinventory": false,
                "cienablueplanet": false,
                "ciscoaci": false,
                "contrail": false,
                "dns": false,
                "docker": false,
                "dynatrace": false,
                "file": false,
                "googlecloud": false,
                "ibmcloud": false,
                "itnm": false,
                "jenkins": false,
                "junipercso": false,
                "kubernetes": true,
                "newrelic": false,
                "openstack": false,
                "rancher": false,
                "rest": false,
                "servicenow": false,
                "taddm": false,
                "vmvcenter": false,
                "vmwarensx": false,
                "zabbix": false
              },
              "shareCassandra": false,
              "storageClassCassandraBackupTopology": "",
              "storageClassCassandraDataTopology": "",
              "storageClassElasticTopology": "",
              "storageClassFileObserver": "",
              "storageSizeCassandraBackupTopology": "50Gi",
              "storageSizeCassandraDataTopology": "50Gi",
              "storageSizeElasticTopology": "75Gi",
              "storageSizeFileObserver": "5Gi"
            },
            "version": "1.6.2"
          }
        },
...
```

Creating an instance NOI, yaml view shows (note clusterDomain missing):
```
apiVersion: noi.ibm.com/v1beta1
kind: NOI
metadata:
  name: example-noi
  labels: {}
  namespace: noioper
spec:
  license:
    accept: false
  advanced:
    antiAffinity: false
    imagePullPolicy: IfNotPresent
    imagePullRepository: cp.icr.io/cp/noi
  serviceContinuity:
    continuousAnalyticsCorrelation: false
    enableAnalyticsBackups: false
    isBackupDeployment: false
  ldap:
    port: '389'
    mode: standalone
    userFilter: 'uid=%s,ou=users'
    bindDN: 'cn=admin,dc=mycluster,dc=icp'
    sslPort: '636'
    url: 'ldap://localhost:389'
    suffix: 'dc=mycluster,dc=icp'
    groupFilter: 'cn=%s,ou=groups'
    baseDN: 'dc=mycluster,dc=icp'
    storageSize: 1Gi
    serverType: CUSTOM
  topology:
    enabled: true
    storageSizeCassandraDataTopology: 50Gi
    storageSizeElasticTopology: 75Gi
    shareCassandra: false
    storageSizeFileObserver: 5Gi
    appDisco:
      enabled: false
      sssStorageSize: 5Gi
      pssStorageSize: 5Gi
      dsStorageSize: 5Gi
      secure: false
    observers:
      docker: false
      taddm: false
      servicenow: false
      ibmcloud: false
      alm: false
      contrail: false
      cienablueplanet: false
      kubernetes: true
      bigfixinventory: false
      junipercso: false
      dns: false
      itnm: false
      ansibleawx: false
      ciscoaci: false
      azure: false
      rancher: false
      newrelic: false
      vmvcenter: false
      rest: false
      appdynamics: false
      jenkins: false
      zabbix: false
      file: false
      googlecloud: false
      dynatrace: false
      aws: false
      openstack: false
      vmwarensx: false
    storageSizeCassandraBackupTopology: 50Gi
    netDisco: false
  ingress:
    class: nginx
    enabled: false
    prefix: nginx.ingress.kubernetes.io
  version: 1.6.2
  persistence:
    storageSizeNCOBackup: 5Gi
    enabled: false
    storageSizeNCOPrimary: 5Gi
    storageSizeImpactServer: 5Gi
    storageSizeKafka: 50Gi
    storageSizeCassandraBackup: 50Gi
    storageSizeZookeeper: 5Gi
    storageSizeCouchdb: 5Gi
    storageSizeCassandraData: 50Gi
    storageSizeDB2: 5Gi
    storageSizeImpactGUI: 5Gi
    storageSizeElastic: 75Gi
  deploymentType: trial
```

Creating NOI instance, form view (note Cluster Domain is there):
<img width="1440" alt="Screen Shot 2020-08-21 at 10 15 42 AM" src="https://user-images.githubusercontent.com/6558437/90907016-0bc1b600-e398-11ea-88f9-65dc69b7651a.png">

When adding a default value to clusterDomain in the example yaml in the csv, it then appears.  I tried the running a local version of the console against the same cluster on release-4.5 branch and it worked fine, so not sure why on OCP 4.5.5 / stable-4.5 channel (and other 4.5.x versions) have this problem.

<img width="708" alt="Screen Shot 2020-08-21 at 10 21 46 AM" src="https://user-images.githubusercontent.com/6558437/90907205-4af00700-e398-11ea-9e3f-314d8d8d784a.png">

**Comment by spadgett** [comment]
@TheRealJon Is this fixed by #5988? (That fix would be available in 4.5.6.)

**Comment by bfabec** [comment]
I believe this is fixed in 4.5.6, thanks!
